pyfar/classes/audio.py

```diff
@@ -108,6 +108,28 @@ def reshape(self, newshape):
 
         return reshaped
 
+    def transpose(self, *axes):
+        """Return a transposed copy of the audio object.
```

Comment:
Maybe even more explicit
```suggestion
        """Transpose time/frequency data and return copy of the audio object.
```